pages/linux/retroarch.md

```diff
@@ -0,0 +1,29 @@
+# retroarch
+
+> RetroArch is a frontend for emulators, game engines and media players.
+> It is the reference implementation of libretro API.
+> More information: <https://github.com/libretro/RetroArch>.
+
+- Start in the menu mode:
+
+`retroarch`
+
+- Start in the full screen mode:
+
+`retroarch --fullscreen`
+
+- List all compiled features:
+
+`retroarch --features`
+
+- Set the path of a configuration file:
+
+`retroarch --config={{path/to/config_file}}`
+
+- Display the help:
```

Comment:
```suggestion
- Display help:
```

Reply:
I don't think it's grammatically more correct without any article. 🤔

Reply:
> I don't think it's grammatically more correct without any article. thinking

I guess both will work. In this case, I think using without article would be fine.